docs/linux-requirements.md

```diff
@@ -0,0 +1,62 @@
+
+# Linux
+
+## Requirements to build native modules
+
```

Comment:
You should add a note that the instructions are for Debian-derived distros, and other distros should adapt them as appropriate.